muffnn/tests/test_mlp_classifier.py

```diff
@@ -177,3 +178,33 @@ def test_replicability():
     probs1 = clf.fit(iris.data, target).predict_proba(iris.data)
     probs2 = clf.fit(iris.data, target).predict_proba(iris.data)
     assert_array_almost_equal(probs1, probs2)
+
+
+def test_partial_fit():
+    X, y = iris.data, iris.target
```

Comment:
Some people would break this test up into finer sections so that it is easier to track down failures. I don't care that much, but if you do, here is a good place to do that.